app/CurrikiGo/Canvas/Client.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace App\CurrikiGo\Canvas;
+
+use App\Models\CurrikiGo\LmsSetting;
+use App\CurrikiGo\Canvas\Contracts\Command;
+use App\Repositories\CurrikiGo\LmsSetting\LmsSettingRepository;
+use App\Repositories\CurrikiGo\LmsSetting\LmsSettingRepositoryInterface;
+use App\Http\Resources\V1\CurrikiGo\LmsSettingResource;
+
+class Client
+{
+    private $lmsSetting;
+    const CURRIKI_MODULE_NAME = 'Curriki Playlists';
+    const API_VERSION = 'v1';
```

Comment:
Please separate private variable and const with blank line.
And normally define the const on the very top of the class.